### ISSUE TITLE
mbar_coul.fit gives pymbar.utils.ParameterError

### ISSUE THREAD

**Issue by xiki-tempula** [alchemistry/alchemlyb#105]
I'm a user of `alchemical-analysis` and recently moving to `alchemlyb`. However, I'm having trouble getting mbar estimator working. 
The sample file is https://drive.google.com/file/d/16bJGCEflqEm5xZhfOQbZ4TSNhorfob4m/view?usp=sharing
If I run it with `alchemical-analysis` with `alchemical_analysis.py -d ./ -p dhdl_ -f 10 -t 310 -s 0 -u kcal -w -g -o ./`. It works fine.
However, if I use the mbar estimator in `alchemlyb`. I got:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/XXX/miniconda3/envs/flamel/lib/python3.8/site-packages/alchemlyb/estimators/mbar_.py", line 90, in fit
    out = self._mbar.getFreeEnergyDifferences(return_theta=True)
  File "/Users/XXX/miniconda3/envs/flamel/lib/python3.8/site-packages/pymbar/mbar.py", line 541, in getFreeEnergyDifferences
    Theta_ij = self._computeAsymptoticCovarianceMatrix(
  File "/Users/XXX/miniconda3/envs/flamel/lib/python3.8/site-packages/pymbar/mbar.py", line 1679, in _computeAsymptoticCovarianceMatrix
    check_w_normalized(W, N_k)
  File "/Users/XXX/miniconda3/envs/flamel/lib/python3.8/site-packages/pymbar/utils.py", line 358, in check_w_normalized
    raise ParameterError(
pymbar.utils.ParameterError: Warning: Should have \sum_n W_nk = 1.  Actual column sum for state 0 was 0.000092. 11 other columns have similar problems
```

The code to reproduce the behavior is:
```
from alchemlyb.parsing.gmx import extract_u_nk
import pandas as pd
u_nk = pd.concat([extract_u_nk('dhdl_{}.xvg'.format(i), T=310) for i in range(11)])
from alchemlyb.estimators import MBAR
MBAR().fit(u_nk)
```

**Comment by orbeckst** [comment]
Do you have a way to see what the `u_nk` in alchemical-analysis contains vs the one in alchemlyb? The estimator should be the same. Or do you have different versions of pymbar?

**Comment by xiki-tempula** [comment]
@orbeckst I'm sorry that the alchemical-analysis is a bit hard to follow. I think what the alchemical-analysis does is not a matter of concern here as one would expect the calculation to run. The pymbar version used here is 3.0.5 and the one used with alchemical-analysis is 3.0.dev3.
